core/src/test/scala/org/apache/spark/memory/UnifiedMemoryManagerSuite.scala

```diff
@@ -309,6 +309,15 @@ class UnifiedMemoryManagerSuite extends MemoryManagerSuite with PrivateMethodTes
     mm.invokePrivate[Unit](assertInvariants())
   }
 
+  test("invalid MEMORY_OFFHEAP_SIZE") {
+    val conf = new SparkConf()
+      .set(MEMORY_OFFHEAP_SIZE, 0)
```

Comment:
Although this is consistent with the other test cases in this test suite, `val conf = new SparkConf().set(MEMORY_OFFHEAP_SIZE, 0)` might be enough for this short test.

Reply:
Thanks, Dongjoon. Removed the test case since you mentioned that there is already another `.checkValue` itself.